api/src/lib/auth.js

```diff
@@ -7,9 +7,13 @@
 
 // @ts-check
 
-import { AuthenticationError } from '@redwoodjs/api'
+import { AuthenticationError } from '@redwoodjs/graphql-server'
 import { logger } from './logger'
 
+import admin from 'firebase-admin'
+
+admin.initializeApp({ projectId: process.env.FIREBASE_PROJECT_ID })
```

Comment:
Still not sure where to put this firebase admin initialization, but I did pair down the config to only need PROJECT_ID.
A bit perplexed that no service account credentials were necessary since the documentation (https://firebase.google.com/docs/auth/admin/verify-id-tokens) for verifyIdToken states that you need to have a service account, but this works even without credentials from what I can tell.